tools/test-launcher.sh

```diff
@@ -6,10 +6,10 @@ set -o pipefail
 cleanup()
 {
     echo "Final clean up"
-    if [ -s out/log/express.log ]; then
-        cat out/log/express.log
-        cat out/log/mock-server.log
-    fi
+#    if [ -s out/log/express.log ]; then
```

Comment:
I guess you got fed up with the huge log dump too :laughing:

I noticed it when trying to run tests locally too.

Reply:
Yes, They are in the build artifact anyway.